packages/js/src/general/components/task.js

```diff
@@ -61,6 +62,8 @@ export const Task = ( { title, id, how, why, duration, priority, isCompleted, ca
 			taskId={ id }
 			callToAction={ callToActionProps }
 			isLoading={ status === ASYNC_ACTION_STATUS.loading }
+			isError={ status === ASYNC_ACTION_STATUS.error }
```

Comment:
When the task is errored and then closed and re-opened, it still shows an error.

I think we should improve that, but as a nice to have for the future, I have created [a separate task](https://github.com/Yoast/reserved-tasks/issues/887) for that.